aws/logs_monitoring/template.yaml

```diff
@@ -13,9 +13,8 @@ Parameters:
     Description: The Datadog API key, which can be found from the APIs page (/account/settings#api). It will be stored in AWS Secrets Manager securely. If DdApiKeySecretArn is also set, this value is ignored.
   DdApiKeySecretArn:
     Type: String
-    AllowedPattern: "arn:.*:secretsmanager:.*"
```

Comment:
We still need to keep the arn pattern as it's used as resource reference when creating the forwarder policy. see https://github.com/DataDog/datadog-serverless-functions/blob/a76e0088087f0a9ce4ee526ffe267893ddc12da5/aws/logs_monitoring/template.yaml#L614